.github/workflows/ci.yml

```diff
@@ -310,8 +310,8 @@ jobs:
         run: |
           prefix="custom-payment-flow/client/android-kotlin"
 
-          if [ "${{ github.ref }}" = "refs/heads/main" ] && \
```

Comment:
It's irrelevant to the PR's description, but `&&` is not intended here in the first place. Fixed to `||`.